### PR TITLE
Reworks aut'akh augments

### PR DESCRIPTION
**Pull request by Alberyk**
This pr changes all aut'akh augments and adds a couple of new ones, with effects and the like. Requested by sleepy.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm [hunk 1]
```diff
@@ -140,26 +140,37 @@
 	sort_category = "Xenowear - Unathi"
 	flags = GEAR_HAS_DESC_SELECTION
 
-/datum/gear/augment/anchor
+/datum/gear/augment/autakh
 	display_name = "soul anchor"
 	description = "An implant that connects the Aut'akh to the Mesh."
```
Comment (SleepyWolfe) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm:145]: change to 
"A rune inscribed mirror or piece of glass placed behind the eyes. Believed to be the 'Window to the Soul' and house the concentrated spirit of an individual."

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm [hunk 2]
```diff
@@ -140,26 +140,37 @@
...
-/datum/gear/augment/haemodynamic
-	display_name = "haemodynamic control system"
-	description = "An Aut'akh augment that allows the user to stabilize themselves."
-	path = /obj/item/organ/internal/augment/haemodynamic
-	cost = 4
-	whitelisted = list("Unathi")
-	sort_category = "Xenowear - Unathi"
+/datum/gear/augment/autakh/calf_override
+	display_name = "calf overdrive"
+	description = "An Aut'akh augment that allows the user to run at high speeds without the cost of stamine, causes damage to the lower body when used."
```
Comment (SleepyWolfe) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm:153]: *stamina here

File: code/__defines/mobs.dm [hunk 3]
```diff
@@ -127,6 +127,7 @@
 #define BP_AUG_FUEL_CELL    "integrated fuel cell"
 #define BP_AUG_AIR_ANALYZER "integrated air analyzer"
 #define BP_AUG_LANGUAGE     "integrated language processor"
+#define BP_AUG_CALF_OVERRIDE     "integrated language processor"
```
Comment (NonQueueingMatt) [inline code/__defines/mobs.dm:130]: 🤔

File: code/modules/organs/subtypes/autakh.dm [hunk 4]
```diff
@@ -67,192 +67,181 @@
...
-	if(owner.reagents)
-		owner.reagents.add_reagent("coagulant", 15)
+	if(!.)
+		return FALSE
 
-/obj/item/organ/internal/augment/haemodynamic/do_broken_act()
-	if(owner.reagents)
-		owner.vessel.remove_reagent("blood",rand(50,75))
-	return TRUE
+	owner.visible_message("<b>[user]'s</b> eyes loudly as they focus ahead.")
```
Comment (NonQueueingMatt) [inline code/modules/organs/subtypes/autakh.dm:181]: loudly what?

File: code/modules/organs/subtypes/autakh.dm [hunk 5]
```diff
@@ -67,192 +67,181 @@
...
 
-/obj/item/organ/internal/augment/adrenal/do_bruised_act()
-	if(owner.reagents)
-		owner.reagents.add_reagent("toxin", 10)
-	return FALSE
+		F.take_damage(10)
+
+		to_chat(owner, "<span class='notice'>\The [src] activates, releasing a stream of chemicals into your veins!</span>")
 
+		owner.reagents.add_reagent("Adrenaline", 15)
```
Comment (NonQueueingMatt) [inline code/modules/organs/subtypes/autakh.dm:141]:
```suggestion
		owner.reagents.add_reagent("adrenaline", 15)
```

**Pull request merged**